sproxy_test.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sproxy_test
+
+import (
+	"encoding/json"
+	"log"
+	"math/rand"
+	"testing"
+	"time"
+
+	sproxy "github.com/GoogleCloudPlatform/stackdriver-reverse-proxy"
+)
+
+func TestReporting(t *testing.T) {
```

Comment:
It might be good to have some benchmarks for the MetricsReporter, could be a follow-up PR.

Reply:
Yeah, for sure.